Automattic/woocommerce-payments
# Add support for Bulgaria

Fixes #5970 

#### Changes proposed in this Pull Request
Add support for Bulgaria

To test the fees, PR `3337` is required on the server side.

#### Testing instructions

**Testing Base fees**
- Go to `WooCommerce` -> `Settings` -> `General`, and set the store Country to Bulgaria and currency to BGN
- Place an order and verify the base fees are applied

<img width="418" alt="Screenshot 2023-05-04 at 12 01 48" src="https://user-images.githubusercontent.com/8667118/236159027-db60047e-db90-45c0-904f-13f75859097a.png">

**Testing International and FX fees**
- Go to `WooCommerce` -> `Settings` -> `General`, and set the store currency to EUR
- Place an order and verify the fees are applied.

<img width="397" alt="Screenshot 2023-05-04 at 12 01 31" src="https://user-images.githubusercontent.com/8667118/236159276-eb3980c4-c69a-44fa-ba9d-b188e6998a9f.png">

**Testing EPS**
- Place an order paid with EPS and verify the fees are applied.
<img width="417" alt="Screenshot 2023-05-04 at 12 11 53" src="https://user-images.githubusercontent.com/8667118/236161246-08d771e3-bcb9-4980-8d0c-5270102e166a.png">



-------------------


**Post merge**
